lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -178,7 +178,7 @@ def send_request(packet, timeout = self.response_timeout)
     if timeout.nil?
       return nil
```

Comment:
This doesn't seem quite right to my eyes, but leaving as-is for now

https://github.com/rapid7/metasploit-framework/pull/6304/files#diff-4d9065091d9752ea933fd50d8b04adab955b22b34466a445398ce1a85dd57c84R201-R203